include/oboe/AudioStreamBase.h

```diff
@@ -105,6 +106,18 @@ class AudioStreamBase {
      */
     int32_t getDeviceId() const { return mDeviceId; }
```

Comment:
I am concerned about storing information in two places.
Can we get rid of mDeviceId and define:

```
int32_t getDeviceId() const {
    return mDeviceIds.empty() ? kUnspecified :  mDeviceIds[0];
}
```

Reply:
Great idea! Done!